extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DatabaseDefaultSetupConfig.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.datasource.deployment.spi;
+
+public interface DatabaseDefaultSetupConfig {
+
+    String DEFAULT_DATABASE_USERNAME = "quarkus";
+
+    // mssql container enforces a 'strong' password with min 8 chars, upper/lowercase, number or special char
+    // db2 derives the instance name from this. Enforces max 8 chars, only lowercase, picky on special chars
+    String DEFAULT_DATABASE_PASSWORD = "quarku_1";
+    String DEFAULT_DATABASE_NAME = "quarkus";
```

Comment:
We don't use the constant in interface pattern. This needs to be a final class with a private constructor and public fields.